modules/apps/commerce/commerce-site-initializer/commerce-site-initializer/src/main/resources/site-initializer/layout-page-templates/master-pages/liferay-commerce-master/page-definition.json

```diff
@@ -0,0 +1,365 @@
+					}
+				},
+				"pageElements": [
+					{
+						"definition": {
+							"htmlProperties": {
+								"htmlTag": "Header"
+							},
+							"indexed": true,
+							"layout": {
+							}
+						},
+						"pageElements": [
+							{
+								"definition": {
+									"fragment": {
+										"key": "header-user",
+										"siteKey": "[$GROUP_KEY$]"
+									},
+									"fragmentConfig": {
+										"showAccountMenu": true,
+										"showCurrencyMenu": false,
+										"showLanguageMenu": true
+									},
+									"fragmentFields": [
+									],
+									"indexed": true
+								},
+								"type": "Fragment"
+							},
+							{
+								"definition": {
+									"fragment": {
+										"key": "header-main",
+										"siteKey": "[$GROUP_KEY$]"
+									},
+									"fragmentConfig": {
+									},
+									"fragmentFields": [
+									],
+									"indexed": true
+								},
+								"type": "Fragment"
+							},
+							{
+								"definition": {
+									"fragment": {
+										"key": "header-nav",
+										"siteKey": "[$GROUP_KEY$]"
+									},
+									"fragmentFields": [
+									],
+									"indexed": true
+								},
+								"pageElements": [
+									{
+										"definition": {
+										},
+										"pageElements": [
+											{
+												"definition": {
+													"indexed": true,
+													"layout": {
+													}
+												},
+												"pageElements": [
+													{
+														"definition": {
+															"fragment": {
+																"key": "com.liferay.fragment.renderer.menu.display.internal.MenuDisplayFragmentRenderer"
+															},
+															"fragmentConfig": {
+																"displayStyle": "horizontal",
+																"hoveredItemColor": "",
+																"selectedItemColor": "",
+																"sublevels": "-1"
+															},
+															"fragmentFields": [
+															],
+															"indexed": true
+														},
+														"type": "Fragment"
+													}
+												],
+												"type": "Section"
+											}
+										],
+										"type": "FragmentDropZone"
+									}
+								],
+								"type": "Fragment"
+							}
+						],
+						"type": "Section"
+					},
+					{
+						"definition": {
+							"cssClasses": [
+								"commerce-main-body",
+								"flex-fill",
+								"pb-5"
+							],
+							"htmlProperties": {
+								"htmlTag": "Main"
+							},
+							"indexed": true,
+							"layout": {
+								"widthType": "Fixed"
+							}
+						},
+						"pageElements": [
+							{
+								"definition": {
+									"fragmentSettings": {
+										"unallowedFragments": [
+										]
+									}
+								},
+								"type": "DropZone"
+							}
+						],
+						"type": "Section"
+					},
+					{
+						"definition": {
+							"htmlProperties": {
+								"htmlTag": "Footer"
+							},
+							"indexed": true,
+							"layout": {
+							}
+						},
+						"id": "5844652d-be2f-1117-5393-227a8ebf1cc5",
```

Comment:
What is this for? is it correct to be in this format?

Reply:
Exports left-overs, I can remove them, though they are ineffective.